src/webgpu/web_platform/util.ts

```diff
@@ -29,6 +29,77 @@ export const kVideoInfo = /* prettier-ignore */ makeTable(
 } as const);
 export type VideoName = keyof typeof kVideoInfo;
 
+// Expectation values about converting video contents to sRGB color space.
+// Source video color space affects expected values.
+// The process to calculate these expected pixel values can be found:
+// https://github.com/gpuweb/cts/pull/2242#issuecomment-1430382811
+const kBt601Red = new Uint8Array([248, 36, 0, 255]);
+const kBt601Green = new Uint8Array([64, 252, 0, 255]);
```

Comment:
Address this comment in a simple way. I think maybe some type like `kBt601PixelValue<Unorm8>.red` would be ideal. But I don't know how to achieve this :(

Reply:
It's not possible for a type parameter to influence a value, because types are purely compile-time constructs that get removed when generating JavaScript. So instead it would have to look something like `kBt601PixelValue.unorm8.red`. I think your code works better than that would.